src/routes/api/index.js

```diff
@@ -1,18 +1,7 @@
-const router = require("express").Router();
+import express from 'express';
+import users from './users';
 
-router.use("/", require("./users"));
+const router = express.Router();
+router.use('/', users);
```

Comment:
This will make all endpoints go to users. `/auth` for authentication and `/user/ for other user routes is better. Also, the route files should be descriptive as such `authRoute` and `userRoute.